craftassist/test/test_conditions.py

```diff
@@ -100,19 +94,21 @@ def test_move_till_condition(self):
         self.assertLessEqual(time_elapsed, 130)
         self.assertGreaterEqual(time_elapsed, 110)
 
-        # TODO make sure cow is moving in x positive direction from below 5 when the test starts
-        # it is now if everything else works, but should force it
-        d = deepcopy(
-            STOP_CONDITION_COMMANDS["follow the cow for 18 seconds after it has x greater than 5"]
-        )
-        start_time = self.agent.get_time()
-        self.handle_logical_form(d, max_steps=5000)
-        end_time = self.agent.get_time()
-        time_elapsed = (end_time - start_time) / TICKS_PER_SEC
-        self.assertEqual(cow_move_sequence[self.agent.world.count - 1 - 18][0][0], 6)
-        self.assertLessEqual(
-            abs(self.agent.pos[0] - cow.pos[0]) + abs(self.agent.pos[2] - cow.pos[2]), 1.01
-        )
+
+#        # FIXME!!!!  this test is flaky....
+#        # TODO make sure cow is moving in x positive direction from below 5 when the test starts
+#        # it is now if everything else works, but should force it
+#        d = deepcopy(
+#            STOP_CONDITION_COMMANDS["follow the cow for 18 seconds after it has x greater than 5"]
+#        )
+#        start_time = self.agent.get_time()
+#        self.handle_logical_form(d, max_steps=5000)
+#        end_time = self.agent.get_time()
+#        time_elapsed = (end_time - start_time) / TICKS_PER_SEC
+#        self.assertEqual(cow_move_sequence[self.agent.world.count - 1 - 18][0][0], 6)
+#        self.assertLessEqual(
```

Comment:
uncomment if not intentional.

Reply:
the test is actually flaky.  will make a task to fix it